packages/edit-widgets/src/store/selectors.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * External dependencies
+ */
+import { toArray } from 'lodash';
+
+export function getWidgetAreas( state ) {
+	return toArray( state.widgetAreas );
```

Comment:
This generates a new array on each call ( a new render ), can we reshape the reducer to avoid that. Otherwise, we'll have to memoize that selector.